### PR TITLE
Fix JSAssemble tool to generate the output file properly

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [ ] `ReleaseNotes` has added or is not needed.
* [ ] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [ ] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Fix the JSAssemble tool
   * Regarding the Region data, the should be generate should be with und_ . because the loader look looking for that
   * e.g) `ilib.data.localeinfo_und_IR` , but curently it generates `ilib.data.localeinfo_IR`
* Added assemble.standard.flutter target to generate ilib-standard-flutter.js file  
  * Choose some features manually (Date/Time Format, NumberFormat, LocacleInfo.. etc)  with webOS locales

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

**Review by ehoogerbeets — COMMENTED** [review]
Let's wait to merge this one until your investigation is further.

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Release Notes for Version 14
 ============================
 
+Build 030
+-------
+Published as version 14.21.0
+
+New Features:
+* Updated to generate the assemed js file by JSAssembl java tool code. This file will work on [QuickJS](https://bellard.org/quickjs/) Engine used by [flutter-js](https://pub.dev/packages/flutter_js)
```
[the remark this answers is not in the document]
Comment (gouniLee) [reply docs/ReleaseNotes.md]: fixed

**Comment by gouniLee** [comment]
> Let's wait to merge this one until your investigation is further.

Yes, We can leave it until the solution is fixed.

**Comment by gouniLee** [comment]
Closing this PR. To replace the `JSAssemble tool`, the code has been updated here instead: https://github.com/iLib-js/ilib-assemble.

**Pull request closed without merging**